clusterloader2/pkg/chaos/nodes.go

```diff
@@ -124,7 +124,16 @@ func (k *NodeKiller) kill(nodes []v1.Node) {
 			time.Sleep(time.Duration(k.config.SimulatedDowntime))
 
 			klog.Infof("%s: Rebooting %q to repair the node", k, node.Name)
-			err = util.SSH("sudo reboot", &node, nil)
+			// Scheduling a reboot in one second, then disconnecting.
+			//
+			// Bash command explanation:
+			// 'nohup' - Making sure that end of SSH connection signal will not break sudo
+			// 'sudo' - Elevated priviliages, required by 'reboot'
+			// 'reboot' - Rebooting node
+			// '+1s' - Parameter to 'reboot', to wait 1 second before rebooting.
```

Comment:
I might be wrong but I'm not sure whether reboot accepts such parameter. I checked `sudo reboot --help` on COS system and such feature wasn't documented anywhere.

Reply:
You are right, this should use `shutdown` command.